src/v2/es-utils.js

```diff
@@ -181,32 +200,20 @@ class ESClass {
       }
       return true;
     } catch (err) {
-      this.logger.debug('Error in kbImport: ' + err.message);
+      this.logElastic('error', `Error in kbImport: ${err.message}`);
       return false;
     }
   }
 
   async checkEsResponse(response, job, item) {
```

Comment:
Is there a more meaningful name than `checkEsResponse`? Also, this code treats the casing of "ES" and "Es" inconsistently. (eg: `checkES`, `runES` - favor consistency).

Reply:
changed it to `checkImportErrors` - that sound better?